### PR TITLE
IT IS IN THE GENES!

### PR DESCRIPTION
**Pull request by J35P312**
added the stringtie -A flag to create a per geneTPM and FPKM tab file (in addition to the gft)

### CODE REVIEW

**Review by jemten — COMMENTED** [review]
Excellent! Now that we have more outputs from stringtie I think you need to update line 411 to `gffcompare(stringtie.out.gtf, ch_gtf)` in order to make the test pass

File: main.nf [hunk 1]
```diff
@@ -406,8 +407,9 @@ workflow {
     picard_collectrnaseqmetrics(ch_indexed_bam, ch_refflat)
 
     // Assemble transcripts
-    stringtie(ch_indexed_bam, ch_gtf, ch_tab)
-    gffcompare(stringtie.out, ch_gtf)
+    stringtie(ch_indexed_bam, ch_gtf)
+    gffcompare(stringtie.out, ch_gtf,ch_tab)
```
Comment (jemten) [inline main.nf:411]:
```suggestion
    gffcompare(stringtie.out.gtf, ch_gtf)
```

Comment (jemten) [reply main.nf:411]: We're not using the tab file in gff compare right?

Comment (J35P312) [reply main.nf]: Currently the tab is not used anywhere =P.
But it is a very nice to have output!

Comment (jemten) [reply main.nf]: Indeed!

**Review by jemten — APPROVED** [review]
Ace!

**Pull request merged**